bin/start-minikube.sh

```diff
@@ -12,16 +12,16 @@ if [ "$unamestr" == "Darwin" ];
 then
   minikube start -p "$PROFILE_NAME" \
   --memory="$MEMORY" \
-  --driver=hyperkit \
+  --driver=virtualbox \
```

Comment:
Any reason to not use `hyperkit` ? For instance I use `hyperkit` on my mac to solve network issue with my minikube.

Reply:
I had Virtualbox. Will switch to hiperkit (also locally) :)